editor/src/document/document_message_handler.rs

```diff
@@ -96,6 +98,11 @@ impl DocumentsMessageHandler {
 			responses.push_back(DocumentMessage::LayerChanged(layer.clone()).into());
 		}
 	}
+
+	// Returns an iterator over the open documents in order
+	pub fn ordered_document_iterator(&self) -> impl Iterator<Item = &DocumentMessageHandler> {
+		self.document_ids.iter().map(|id| self.documents.get(id).expect("document id is not found in the document hashmap"))
```

Comment:
@Keavon this sounds a bit off, shouldn't it be `was not found` or `is not in`?

Reply:
Good catch